src/portals/hope/features/claims/claim-details/ClaimSubclaims/ClaimSubclaims.tsx

```diff
@@ -90,7 +99,8 @@ const Subclaim: React.FC<{
   subclaim: ClaimInformationSubclaimFragment
   memberId: string
   claimId: string
-}> = ({ subclaim, memberId, claimId }) => {
+  state?: ClaimState
```

Comment:
ClaimState should always exist on a claim
```suggestion
  state: ClaimState
```